eng/Update-PkgVersion.ps1

```diff
@@ -44,59 +44,23 @@ Param (
     [string] $PackageDirName,
     [string] $NewVersionString
 )
-# Regular expression as specified in https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
-$VERSION_REGEX = "^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$"
+
+. ${PSScriptRoot}\common\scripts\SemVer.ps1
 
 # Updated Version in csproj and changelog using computed or set NewVersionString
 function Update-Version($Unreleased=$True, $ReplaceVersion=$False)
```

Comment:
Should take in the SemVer as an argument.

Reply:
done